app/formSchema/uiSchema/uiSchema.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  acknowledgements,
+  alternateContact,
+  authorizedContact,
+  benefits,
+  budgetDetails,
+  contactInformation,
+  coverage,
+  estimatedProjectEmployment,
+  existingNetworkCoverage,
+  organizationLocation,
+  organizationProfile,
+  otherFundingSources,
+  projectArea,
+  projectInformation,
+  projectFunding,
+  projectPlan,
+  submission,
+  supportingDocuments,
+  review,
+  techSolution,
+  templateUploads,
+} from './pages';
+
+const uiSchema = {
+  'ui:order': [
+    ...projectInformation.uiOrder,
+    ...projectArea.uiOrder,
+    ...existingNetworkCoverage.uiOrder,
+    ...budgetDetails.uiOrder,
+    ...projectFunding.uiOrder,
+    ...otherFundingSources.uiOrder,
+    ...techSolution.uiOrder,
+    ...benefits.uiOrder,
+    ...projectPlan.uiOrder,
+    ...estimatedProjectEmployment.uiOrder,
+    ...templateUploads.uiOrder,
+    ...supportingDocuments.uiOrder,
+    ...coverage.uiOrder,
+    ...organizationProfile.uiOrder,
+    ...organizationLocation.uiOrder,
+    ...contactInformation.uiOrder,
+    ...authorizedContact.uiOrder,
+    ...alternateContact.uiOrder,
+    // ...review.uiOrder,
+    ...acknowledgements.uiOrder,
+    ...submission.uiOrder,
+  ],
+  'ui:title': '',
+  ...projectInformation,
```

Comment:
```suggestion
  projectInformation,
```
UI schema should match the shape of the data, instead of flattening the objects.
In `ApplicationForm`, you'll want to always use `uiSchema[sectionName]`, instead of the current behaviour